game.rb

```diff
@@ -0,0 +1,27 @@
+require './item'
+
+# class for creating game object for items collection
+class Game < Item
+  attr_accessor :publisher, :cover_state
+  attr_reader :last_played, :multiplayer
+
+  @games = []
+  def initialize(multiplayer:, last_played:, **args)
+    super(**args)
+    @multiplayer = multiplayer
+    @last_played = Date.parse(last_played)
+    self.class.all << self
+  end
+
+  def can_be_archived?
+    Date.today.year - @publish_date.year > 10 && Date.today.year - @last_played.year > 2
+  end
```

Comment:
- [ ] As it was pointed out earlier, in order to avoid code repetition, since `can_be_archived` had been implemented in the `superclass` (item.rb); kindly use the super keyword to access the method. 👍 This will show a good comprehension of Inheritance in Ruby.

Reply:
Resolved